packages/shared/notionable-hooks/src/summary/index.ts

```diff
@@ -35,27 +41,27 @@ export const OrderDetailLabels = defineMessages({
 
 export const TradeSummaryLabels = {
```

Comment:
Is there a reason why these changed? They are used in many places, not sure if I changed them or you did...

Reply:
Oh I see, I'd rather use the ones that I have than switch them to what Isaac has in the mocks. Can you revert these changes here?

Reply:
I made these changes in accordance with what was in the new mocks for trade summary table